app/models/metric/ci_mixin.rb

```diff
@@ -1,5 +1,6 @@
 module Metric::CiMixin
   extend ActiveSupport::Concern
+  include SupportsFeatureMixin
```

Comment:
I see, I got away of this `include`.

Instead, I made sure (in first commit) that everything that includes `CiMixin` includes the `SupportsFeatureMixin` as well.